ktor-server/ktor-server-core/jvm/src/io/ktor/sessions/SessionTrackerById.kt

```diff
@@ -36,7 +36,7 @@ class SessionTrackerById<S : Any>(
                 serializer.deserialize(text)
             }
         } catch (notFound: NoSuchElementException) {
-            call.application.log.debug("Failed to lookup session: $notFound")
+            call.application.log.debug("Failed to lookup session: ${notFound.message ?: notFound.toString()}")
```

Comment:
Should be "look up", as opposed to "lookup".

Reply:
notFound.message usually contains something like session ID [number] is not found. I believe it should be enough. So I simply removed exception class name "NoSuchElementException" that was too loud.

Reply:
Note that in production you can't see these message because of the debug log level.

Reply:
Perhaps, we could add something like "The session id is wrong or outdated"

Reply:
But would still need to be the verb "look up".